06/obesenec.py

```diff
@@ -0,0 +1,187 @@
+hangman = [
+        """
+
+
+
+
+
+
+        -----------------
+        """
+        ,
+        """ 
+        +
+        |
+        |
+        |
+        |
+        |
+        ~~~~~~~
+        ------------------
+        """
+        ,
+        """
+        +---.
+        |
+        |
+        |
+        |
+        |
+        ~~~~~~~
+        ------------------
+        """
+        ,
+        """
+        +---.
+        |   |
+        |
+        |
+        |
+        |
+        ~~~~~~~
+        ------------------
+        """
+        ,
+        """
+        +---.
+        |   |
+        |   O
+        |
+        |
+        |
+        ~~~~~~~
+        ------------------
+        """
+        ,
+        """
+        +---.
+        |   |
+        |   O
+        |   |
+        |
+        |
+        ~~~~~~~
+        ------------------
+        """
+        ,
+        """
+        +---.
+        |   |
+        |   O
+        | --|
+        |
+        |
+        ~~~~~~~
+        ------------------
+        """
+        ,
+        """
+        +---.
+        |   |
+        |   O
+        | --|--
+        |
+        |
+        ~~~~~~~
+        ------------------
+        """
+        ,
+        """
+        +---.
+        |   |
+        |   O
+        | --|--
+        |  /
+        |
+        ~~~~~~~
+        ------------------
+        """
+        ,
+        """
+        +---.
+        |   |
+        |   O
+        | --|--
+        |  / \
+        |
+        ~~~~~~~
+        ------------------"""]
+
+import random
+slova = ["projekt", "kvetina", "buchta"]
+
+def vyber_nahodne_slovo(slova):
+    # Vybira nahodne slovo
+    tajne_slovo = random.choice(slova)
+    return tajne_slovo
+
+def start(hangman, spatna_pismena, spravna_pismena, tajne_slovo):
+    print(hangman[len(spatna_pismena)])
+    print()
+
+    print("Spatna pismena:",end=" ")
```

Comment:
Proměnná špatná písmena (i správná písmena) by mohla být spíš seznam. Dávalo by to tak větší smysl a bylo by jednodušší na obsluhu na řídcích 164 a 175, kde by stačilo dát spatna_pismena.append(tip). :slightly_smiling_face: 

Potom by celý zápis 
```
print("Spatna pismena:",end=" ")
for pismeno in spatna_pismena:
        print(pismeno, end=" ")
```
mohl vypadat
```
print("Spatna pismena:", *spatna_pismena)
```

https://stackoverflow.com/questions/3394835/use-of-args-and-kwargs